config/detekt/detekt.yml

```diff
@@ -626,7 +626,7 @@ style:
     active: false
   ReturnCount:
     active: true
-    max: 2
+    max: 3
```

Comment:
Don't change the configuration, if something cannot be refactored to fall under this limit then add a `@Suppress("ReturnCount")` on the method instead.

Reply:
adjusted 👍